content/docs/command-reference/repro.md

```diff
@@ -36,9 +39,9 @@ and caches relevant <abbr>data artifacts</abbr> along the way.
 💡 For convenience, a Git hook is available to remind you to `dvc repro` when
 needed after a `git commit`. See `dvc install` for more details.
 
-There's a few ways to restrict the stages that will be regenerated by this
-command: by specifying stage file `targets`, or by using the `--single-item`,
-`--cwd`, or other options.
+There are a few ways to restrict the stages that will be regenerated by this
+command: by specifying stage or `.dvc` file to reproduce, or by using the
```

Comment:
Is "specifying stage or `.dvc` file to reproduce" correct grammatically? It was "specifying stage file `targets`" before (which is correctly written).

Reply:
> which is correctly written

But stages are not files now so we can't write "stage file `targets`". We have to specify stage and `.dvc` file separately. Maybe we can write "specifying `targets` (stage or .dvc file).."

Reply:
I know we need to change it. What I'm saying is the way you wrote it is incorrect grammatically. Please fix it.

Reply:
> Please fix it.

So is "specifying `targets` (stage or .dvc file), ..." correct?

Reply:
Nope, the fix is much simpler than that. It has to do with singular vs plural... If you use singular then an article is needed (a, the, etc.) yet missing rn. But should it be singular in the first place?

Reply:
Even the idea to introduce  `.dvc` files here doesn't make any sense in the first place. Since a `.dvc` file can only be used as the first node in a branch and hence no stages will be regenerated if we specify `.dvc` file here.

For an example - 

![image](https://user-images.githubusercontent.com/46163196/87843956-2c4cab00-c8d6-11ea-8e26-42a9459e2a31.png)

Reply:
> a .dvc file can only be used as the first node in a branch and hence no stages will be regenerated if we specify .dvc file here.

If the tracked data changes, the .dvc file will become outdated and `repro` will commit the new data to the cache. So it's perfectly valid to use .dvc files here

I think we're over-complicating this discussion. Please just fix the grammar of the current text.